src/firebase-config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
please revert my firebase credentials.

Reply:
I stored it in a separate file so that it will be protected. You need to store it in a firebase-config.json so that it will not be included in this repo